src/main/target/NUCLEOF103RG/config.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "platform.h"
+
+#ifdef USE_TARGET_CONFIG
+
+#include "common/axis.h"
+#include "common/utils.h"
+
+#include "drivers/io.h"
+
+#include "fc/config.h"
+#include "fc/rc_controls.h"
+#include "fc/controlrate_profile.h"
+
+#include "flight/failsafe.h"
+#include "flight/pid.h"
+
+#include "pg/rx.h"
+
+#include "rx/rx.h"
+
+#include "sensors/acceleration.h"
+#include "sensors/compass.h"
+#include "sensors/gyro.h"
+
+#include "pg/beeper_dev.h"
+#include "pg/flash.h"
+#include "pg/motor.h"
+
+#include "hardware_revision.h"
+
+void targetConfiguration(void)
+{
+#ifdef BEEBRAIN
```

Comment:
Not really sure if the complexity of making this 'emulate' different F1 targets is needed, or what this is meant to give us - shouldn't this just be an F1 target of its own, just like the other NUCLEO targets are?

Reply:
Ah, this is just a side effect of copying `target/NAZE/config.c`; deleted.